### PR TITLE
Update comment about all available types under list of allowed types for property type constraints

### PR DESCRIPTION
**Pull request by Hunterness**
Was a bit confusing to first have supported types and then say available types (as available could be seen as 'available to the constraints as well')

### CODE REVIEW

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-899.surge.sh

Updated pages:
- [Examples](https://neo4j-docs-cypher-899.surge.sh/cypher-manual/5/constraints/examples)

File: modules/ROOT/pages/constraints/examples.adoc [hunk 1]
```diff
@@ -1303,7 +1303,7 @@ The allowed property types for the constraints are:
 * `LIST<POINT NOT NULL>` label:new[Introduced in 5.10]
 * Any closed dynamic union of the above types, e.g. `INTEGER | FLOAT | STRING`. label:new[Introduced in 5.11]
 
-For all available Cypher types, see the section on xref::values-and-types/property-structural-constructed.adoc#types-synonyms[types and their synonyms].
+For all types available in Cypher, see the section on xref::values-and-types/property-structural-constructed.adoc#types-synonyms[types and their synonyms].
```
Comment (mnd999) [inline modules/ROOT/pages/constraints/examples.adoc]: Is this better?

```suggestion
For a complete reference describing all types available in Cypher, see the section on xref::values-and-types/property-structural-constructed.adoc#types-synonyms[types and their synonyms].
```

Comment (Hunterness) [reply modules/ROOT/pages/constraints/examples.adoc]: it might be yes 👍

File: modules/ROOT/pages/constraints/examples.adoc [hunk 2]
```diff
@@ -1713,7 +1713,7 @@ The allowed property types for the constraints is:
...
```
Comment (Hunterness) [inline modules/ROOT/pages/constraints/examples.adoc]:
```suggestion
For a complete reference describing all types available in Cypher, see the section on xref::values-and-types/property-structural-constructed.adoc#types-synonyms[types and their synonyms].
```

**Review by mnd999 — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
This PR includes documentation updates
View the updated docs at https://neo4j-docs-cypher-899.surge.sh

Updated pages:
- [Examples](https://neo4j-docs-cypher-899.surge.sh/cypher-manual/5/constraints/examples)

**Review by JPryce-Aklundh — APPROVED** [review]

**Comment by neo-technology-commit-status-publisher** [comment]
Thanks for the documentation updates.

The preview documentation has now been torn down - reopening this PR will republish it.

**Pull request merged**